<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="Priti" failures="0" tests="4" name="Login_Inventory_Test(failed)" time="39.963" errors="0" timestamp="2023-12-24T22:46:33 IST">
  <testcase classname="testCases.Inventory_Page_Test" name="remove2ProductTest" time="4.319"/>
  <testcase classname="testCases.Inventory_Page_Test" name="verifyProductLableTest" time="0.095"/>
  <testcase classname="testCases.LoginPageTest" name="loginToApplication" time="0.253"/>
  <testcase classname="testCases.LoginPageTest" name="verifyURLofApplication" time="0.028"/>
  <testcase classname="testCases.Inventory_Page_Test" name="verifyLinkedInLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLinkedInLogoTest -->
  <testcase classname="testCases.LoginPageTest" name="verifyTitleOfApplication" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTitleOfApplication -->
  <testcase classname="testCases.Inventory_Page_Test" name="verifyFacebookLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyFacebookLogoTest -->
  <testcase classname="testCases.Inventory_Page_Test" name="verifyTwitterLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTwitterLogoTest -->
  <testcase classname="testCases.Inventory_Page_Test" name="add6ProductTest" time="0.0">
    <ignored/>
  </testcase> <!-- add6ProductTest -->
</testsuite> <!-- Login_Inventory_Test(failed) -->
